3.2/root/usr/bin/run-mongod

```diff
@@ -30,6 +30,7 @@ trap 'cleanup' SIGINT SIGTERM
 if [ ! -s $MONGODB_CONFIG_PATH ]; then
   # Generate config file for MongoDB
   envsubst < ${CONTAINER_SCRIPTS_PATH}/mongodb.conf.template > $MONGODB_CONFIG_PATH
+  setup_wiredtiger_cache
```

Comment:
I was modifying template file, but it didn't work - template file is owned by root. Also it is `/usr/share/container-scripts/mongodb/`.

Ok, I modified Dockerfiles to change permissions of template file.